src/test/java/tech/aroma/data/sql/serializers/ReactionsSerializerTest.kt

```diff
@@ -92,7 +91,7 @@ class ReactionsSerializerTest
                 .thenReturn("")
 
         val result = instance.deserialize(row)
-        assertThat(result, notNull)
+        assertThat(result, tech.aroma.data.notNull)
```

Comment:
Why not just import this?
![](https://render.bitstrips.com/v2/cpanel/10220248-288439948_4-s1-v1.png?transparent=1&palette=1)